packages/espressocash_app/lib/features/outgoing_split_key_payments/models/outgoing_split_key_payment.dart

```diff
@@ -14,6 +14,8 @@ class OutgoingSplitKeyPayment with _$OutgoingSplitKeyPayment {
     required CryptoAmount amount,
     required DateTime created,
     required OSKPStatus status,
+    DateTime? linksGeneratedAt,
+    DateTime? resolvedAt,
```

Comment:
You don't need to store `resolvedAt` here, it will be on the `canceled`/`withdrawn` statuses, and they are terminal.